playbooks/roles/l2tp-ipsec/templates/streisand-l2tp-iptables.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Set iptables rules required to L2TP/IPsec gateway
```

Comment:
Typo: "Set iptables rules required _to_ L2TP/IPSec gateway" -> "Set iptables rules required **for** L2TP/IPSec gateway"

Reply:
Fixed.